### PR TITLE
uses w16 instead of w16microbase

### PR DESCRIPTION
**Pull request by scriptnull**
https://github.com/Shippable/admiral/issues/2085

### CODE REVIEW

File: common/scripts/configs/create_master_runtime_templates.sql [hunk 1]
```diff
@@ -20,9 +20,10 @@ do $$
       end if;
 
       -- x86_64 WindowsServer_2016 Master
+      update "runtimeTemplates" set "defaultTaskImage" = 'w16' where "archTypeCode" = 8000 and "osTypeCode" = 9002 and "version" = 'Master';
```
Comment (ric03uec) [inline common/scripts/configs/create_master_runtime_templates.sql]: needs an if condition

Comment (scriptnull) [reply common/scripts/configs/create_master_runtime_templates.sql]: I guess `where` is an equivalent of `if` here. `where` will only update the rows that satisfies the condition.
```
where "archTypeCode" = 8000 and "osTypeCode" = 9002 and "version" = 'Master';
```

Comment (ric03uec) [reply common/scripts/configs/create_master_runtime_templates.sql]: what if runtime templates is not there for old installations?

Comment (scriptnull) [reply common/scripts/configs/create_master_runtime_templates.sql]: Execution enters this block only if `runtimeTemplates` table is present in the installation ( https://github.com/Shippable/admiral/blob/master/common/scripts/configs/create_master_runtime_templates.sql#L3 )

So, there shouldn't be any problem if the `runtimeTemplates` table does not exist or row for windows is not present in `runtimeTemplates`

**Pull request merged**